client/allocrunner/alloc_runner.go

```diff
@@ -259,7 +259,7 @@ func (ar *allocRunner) Run() {
 			ar.logger.Error("prerun failed", "error", err)
 
 			for _, tr := range ar.tasks {
-				tr.MarkFailedDead(fmt.Sprintf("failed to setup runner: %v", err))
+				tr.MarkFailedDead(fmt.Sprintf("failed to setup alloc: %v", err))
```

Comment:
Sorry this is an unrelated wording improvement. I actually got this error when testing, and it will be shown to users a lot more often when using Connect as the opportunities for AR Prerun failures is increased. "runner" is a really confusing term to put in front of users and "alloc" is synonymous from their perspective.